vms/platformvm/state/stakers_helpers.go

```diff
@@ -0,0 +1,70 @@
+// Copyright (C) 2019-2024, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package state
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/ava-labs/avalanchego/database"
+	"github.com/ava-labs/avalanchego/utils/timer/mockable"
+)
+
+func NextBlockTime(state Chain, clk *mockable.Clock) (time.Time, bool, error) {
```

Comment:
this is pretty annoying. I have moved this methods here from `executor` package. The reason is that I need to use them in `txstest` package ( I need to know next block time to update unit fees), and leaving them in `executor` package would cause an import cycle.
It does not feel very natural to have these methods here, but not currently sure where else should I place them to unlock compilation